content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -83,7 +83,7 @@ The following command will create a Horizontal Pod Autoscaler that maintains bet
 controlled by the php-apache deployment we created in the first step of these instructions.
 Roughly speaking, HPA will increase and decrease the number of replicas
 (via the deployment) to maintain an average CPU utilization across all Pods of 50%
-(since each pod requests 200 milli-cores by `kubectl run`), this means average CPU usage of 100 milli-cores).
+(since each pod requests 200 milli-cores by `kubectl run`, this means average CPU usage of 100 milli-cores).
```

Comment:
@tengqm @nuno-faria This would read better if it were made a separate sentence. 

As in, 

The following command will create a Horizontal Pod Autoscaler that maintains between 1 and 10 replicas of the Pods
controlled by the php-apache deployment we created in the first step of these instructions.
Roughly speaking, HPA will increase and decrease the number of replicas
(via the deployment) to maintain an average CPU utilization across all Pods of 50%.
Since each pod requests 200 milli-cores by `kubectl run`, this means an average CPU usage of 100 milli-cores

Reply:
Agreed, thanks @divya-mohan0209. I pushed a new commit with those changes.